mingw-w64-emacs/PKGBUILD

```
@@ -69,6 +69,8 @@ build() {
 
   # Required for nanosleep with clang
   export LDFLAGS="${LDFLAGS} -lpthread"
+  # -D_FORTIFY_SOURCE breaks build
+  CFLAGS=${CFLAGS//"-Wp,-D_FORTIFY_SOURCE=2"}
```

```suggestion
  CFLAGS+=" -U_FORTIFY_SOURCE"
```

would that work too?